templates/rhsm-katello-reconfigure.erb

```diff
@@ -16,11 +16,13 @@ type -P subscription-manager >/dev/null || type -P subscription-manager-cli >/de
 # backup configuration during the first run
 test -f $CFG_BACKUP || cp $CFG $CFG_BACKUP
 
-#create the cert
+# create the cert and add newline to end of cert
 echo "$KATELLO_SERVER_CA_DATA" > $KATELLO_CERT_DIR/$KATELLO_SERVER_CA_CERT
+echo "" >> $KATELLO_CERT_DIR/$KATELLO_SERVER_CA_CERT
```

Comment:
You can also echo `"$KATELLO_SERVER_CA_DATA\n"` so you don't need a separate command.